compiler/src/dmd/backend/melf.d

```diff
@@ -459,3 +461,141 @@ struct Elf64_Rela
 enum GRP_COMDAT   = 1;
 enum GRP_MASKOS   = 0x0ff0_0000;
 enum GRP_MASKPROC = 0xf000_0000;
+
+/***************************** 64 bit AArch64 *****************************************/
+
+        enum R_AARCH64_NONE                        =    0;
```

Comment:
are these intended to be indented? Otherwise looks good.

Reply:
It matches the indenting of the X86-64 declarations.